test/integration/test_resolvers.py

```diff
@@ -0,0 +1,24 @@
+"""Integration tests for the conda dependency resolution."""
+
+import os
+from base import integration_util
+from base.api import ApiTestCase
+
+GNUPLOT = {u'version': u'4.6', u'type': u'package', u'name': u'gnuplot'}
+
+class CondaResolutionIntegrationTestCase(integration_util.IntegrationTestCase, ApiTestCase):
```

Comment:
So I thought hacking test_render might be a work around - modifying the entry point

```
import svgwrite

import os
from base import integration_util
from base.api import ApiTestCase
```

fixes the problem - modifying as

```
```
import os
from base import integration_util
from base.api import ApiTestCase
import svgwrite
```

Makes the error happen sooner.